### PR TITLE
Create bookings

### PR DESCRIPTION
**Pull request by eleonorakiziv**
1. Create booking model in schema and seed 2 Bookings
2. Add validations to booking, lesson and user models
3. Config routes

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,5 +3,8 @@
   resources :pages, only: :about
   devise_for :users
   resources :users, only: [:show, :index]
-  resources :lessons
+  resources :lessons do
+    resources :bookings, only: [:new, :create]
+  end
+  resources :bookings, except: [:new, :create]
```
Comment (arthurphiladelpho) [inline config/routes.rb]: add index

File: app/models/user.rb [hunk 2]
```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  # Somebody, please, check this. I am not sure
+  has_many :lessons, through: :bookings
```
Comment (arthurphiladelpho) [inline app/models/user.rb]: keep has_many :lessons

Comment (eleonorakiziv) [reply app/models/user.rb]: ok

**Comment by eleonorakiziv** [comment]
You can merge it now.

**Pull request merged**